unabridged-works-volume-4/original/17-letters.adoc

```diff
@@ -651,11 +690,14 @@ but without the true, pure, fresh life.
 
 From a faithful friend and lover of souls.
```

Comment:
SHould this be a [sscl]?

Reply:
I could go either way, but I think lean towards not making it a sscl, like you left it. 👍